File: apps/api/src/Features/TipoDeGasto/Models/TipoDeGastoModel.cs
namespace Features.TipoDeGasto.Models;

public class TipoDeGastoModel
{
    public int IdTipoDeGasto { get; set; }
    public string Nombre { get; set; } = string.Empty;
    public string Descripcion { get; set; } = string.Empty;
    public decimal MontoMaximo { get; set; }
}

File: apps/web/src/app/features/tipo-de-gasto/models/tipo-de-gasto.model.ts
export interface TipoDeGastoCreate {
  idTipoDeGasto: number;
  nombre: string;
  descripcion: string;
  montoMaximo: number;
}

export interface TipoDeGastoRead {
  idTipoDeGasto: number;
  nombre: string;
  descripcion: string;
  montoMaximo: number;
}

export interface TipoDeGastoUpdate {
  idTipoDeGasto: number;
  nombre: string;
  descripcion: string;
  montoMaximo: number;
}
